_data/people.yaml

```diff
@@ -1658,6 +1658,27 @@ sudarshangc:
     expertise:
         - OLS-1 project lead
 
+thessaly:
+    first-name: Julieta
+    last-name: Arancio
+    twitter: cassandreces
+    email: jarancio@unsam.edu.ar
+    website: https://julieta.aranc.io/
+    orcid: 0000-0002-8624-4182
+    expertise:
+       - How to open your science hardware prototypes
+       - How to contribute/use available open science hardware in your research
+       - How to run effective and inclusive citizen/community science projects
+       - Using Wikidata and other linked data tools to boost your research
+       - General panorama of Open science in Latin America
+       - Thinking what 'science and knowledge democratization' means in context
```

Comment:
```suggestion
       hardware prototypes, contributing and using open science hardware, inclusive citizen/community science projects, Wikidata and other linked data tools, Open science in Latin America, 'science and knowledge democratization' in context
```

Reply:
@thessaly is it ok to keep it as keyword-ish?

Reply:
@malvikasharan I quickly reformated your changes as a list of keywords for the rendering on the website. I hope it is ok

Reply:
sure, feel free to adapt!